nequip/train/_key.py

```diff
@@ -11,7 +11,8 @@
 TRAIN = "training"
 
 ABBREV = {
-    AtomicDataDict.TOTAL_ENERGY_KEY: "e",
+    AtomicDataDict.TOTAL_ENERGY_KEY: "E",
+    AtomicDataDict.PER_ATOM_ENERGY_KEY: "e",
```

Comment:
this is important to document in changelog, will confuse people

are longer abbrevs allowed? Ei, Etot or something?

Reply:
```suggestion
    AtomicDataDict.TOTAL_ENERGY_KEY: "e",
    AtomicDataDict.PER_ATOM_ENERGY_KEY: "Ei",
```